src/ansys/aedt/toolkits/antenna/ui/run_frontend.py

```diff
@@ -0,0 +1,197 @@
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import os
+import sys
+
+# isort: off
+
+# Default user interface properties
+from ansys.aedt.toolkits.antenna.ui.models import properties
+
+# isort: on
+
+# PySide6 Widgets
+from PySide6.QtWidgets import QApplication
+from PySide6.QtWidgets import QMainWindow
+
+# Toolkit frontend API
+from actions import Frontend
+from ansys.aedt.toolkits.common.ui.common_windows.home_menu import HomeMenu
+from ansys.aedt.toolkits.common.ui.common_windows.settings_column import SettingsMenu
+
+# Import general common frontend modules
+from ansys.aedt.toolkits.common.ui.logger_handler import logger
+
+# Common windows
+from ansys.aedt.toolkits.common.ui.main_window.main_window_layout import MainWindowLayout
+
+# New windows
+from windows.antenna_catalog.antenna_catalog_menu import AntennaCatalogMenu
+from windows.antenna_results.antenna_results_menu import AntennaResultsMenu
+from windows.antenna_synthesis.antenna_synthesis_menu import AntennaSynthesisMenu
+
+# Windows
+
+
+# Backend URL and port
+if len(sys.argv) == 3:
+    properties.backend_url = sys.argv[1]
+    properties.backend_port = int(sys.argv[2])
+
+url = properties.backend_url
+port = properties.backend_port
+
+os.environ["QT_API"] = "pyside6"
+os.environ["QT_FONT_DPI"] = "96"
+
+if properties.high_resolution:
+    os.environ["QT_SCALE_FACTOR"] = "2"
+
+
+class ApplicationWindow(QMainWindow, Frontend):
+    def __init__(self):
+        super().__init__()
+
+        self.thread = None
+        self.properties = properties
+
+        # General Settings
+
+        # Create main window layout
+        self.ui = MainWindowLayout(self)
+        self.ui.setup()
+
+        # Settings menu
+        self.settings_menu = SettingsMenu(self)
+        self.settings_menu.setup()
+
+        # Check backend connection
+        success = self.check_connection()
+
+        # Populate settings column
+        if not success:
+            msg = "Error getting properties from backend. User interface running without backend"
```

Comment:
```suggestion
            msg = "Error getting properties from backend. User interface is running without backend."
```
message must always have a verb and concluding punctuation for each sentence. Please check them all to ensure that this is the case.